src/agent/sqlite_manager/src/sqlite_manager.cpp

```diff
@@ -2,6 +2,10 @@
 
 namespace sqlite_manager
 {
-    SQLiteManager::SQLiteManager() {}
+    SQLiteManager::SQLiteManager(const std::string& dbName)
+        : m_dbName(dbName)
+        , m_db(make_unique<SQLite::Database>(dbName, SQLite::OPEN_READWRITE | SQLite::OPEN_CREATE))
```

Comment:
I'm surprised `std::` is not necessary for `make_unique`. Anyway, could we add it?

Reply:
Added.